### DISCUSSION TITLE
DOUBT REGARDING FRT VIRTUAL INTERNSHIP

### DISCUSSION THREAD

**Discussion by UtCrushIt** [github/india#523] [Students]
My PC has Intel Celeron processor and 2GB RAM. with that specs would I be able to complete all the courses of FRT virtual internship?  @ey-mentor1 , @ey-mentor2 , @EY-Mentor3 , @EY-Mentor4

**Comment by PRIYA-S-102** [comment]
have you completed your FRT Internship project if yes help me for completing.
mail me at princessrajpriya26@gmail.com
